content/blog/2020-11-18-análise-da-covid-19-em-3-cidades-do-estado-do-paraná.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: Análise da COVID-19 em 3 cidades do estado do Paraná
+author: Marcelo e Davi
+date: '2020-11-18'
+slug: análise-da-covid-19-em-3-cidades-do-estado-do-paraná
+categories:
+  - covid19
+tags:
+  - covid
+  - analises
+description: ''
+images:
+  - 'img/covid/post_2.png'
+---
+
+```{r echo=FALSE,message=FALSE,warning=FALSE}
+library(datacovidbr)
+library(dplyr)
+library(ggplot2)
+library(plotly)
+
+df <- brasilio()
+
+df.cidades <- df%>%filter(df$state =="PR")
+df.cidades.analisadas <- df.cidades %>% filter(df.cidades$city =="Curitiba" |
+                                             df.cidades$city == "Londrina" |
+                                             df.cidades$city == "Cascavel")
+df.cidades.analisadas["deaths_per_100k_inhabitants"] <- (df.cidades.analisadas$deaths/df.cidades.analisadas$estimated_population_2019)*100000
+
+```
+
+# Criação de bandeiras da COVID-19 no estado do Paraná
+
+ O estado do Paraná adotou no mês de junho bandeiras com as cores amarelo, laranja e vermelho tendo como significado respectivamente alerta, risco médio e alto risco. Para protocolar essas cores o estado deve levar em consideração alguns requisitos dividido em dois critérios, propagação da doença e capacidade de atendimento.
+
+# Casos confirmados em Cascavel, Curitiba e Londrina
+
+```{r echo=FALSE,message=FALSE,warning=FALSE}
+# GRAFICO CONFIRMADOS
+graph.confirmados <- df.cidades.analisadas %>% 
+  ggplot(mapping = aes(fill = city,x = date, y = confirmed, color = city,
+                       text = paste0(city, "\n",
+                                     date, "\n", 
+                                     "Confirmados: ", confirmed))) +
+  xlab("Dias/Meses") +
+  ylab("Confirmados") +
+  geom_point(aes(y = confirmed), size = 0.4)
+ggplotly(graph.confirmados, tooltip = "text")
```

Comment:
Os gráficos gerados pelo plotly não estão aparecendo na página. Acho que precisa fazer alguma coisa tipo o que está aqui https://harborisland.github.io/plotly/how-to-use-plot.ly-and-ggplotly-with-R-Markdown para funcionar